### ISSUE TITLE
Module U2NETP doesn't exist. Check if it is installed

### ISSUE THREAD

**Issue by NaeemKhan333** [w-okada/image-analyze-workers#12]
Thank you for helping blog, I have issue related to U2NET model conversion.I am using your blog 

https://dannadori.medium.com/convert-pytorch-model-to-tensorflowjs-fb3bc8e90589

But while I run this command 

python3 /home/user/intel/openvino_2021/deployment_tools/tools/model_downloader/pytorch_to_onnx.py --import-module model.u2net --model-name U2NETP --input-shape 1,3,320,320 --weights saved_models/u2netp/u2netp.pth --output-file u2netp_320x320.onnx --input-names "x" --output-names "a/F.sigmoid(d0)"

I am getting this error
-> I have "model" folder and it contain the 'u2net.py' file .

Module U2NET doesn't exist. Check if it is installed
No module named 'model'

Please guide me about it.Thanks

**Comment by NaeemKhan333** [comment]
@dannadori Yes I have set the 
```
export PYTHONPATH=/U-2-Net
```
But still facing this thing.What could be the problem?

**Comment by NaeemKhan333** [comment]
@dannadori Thanks, I have solved this issue by giving absolute path of the working directory

```
export PYTHONPATH=/home/user/Downloads/U-2-Net
```
